docs/section10/10-9.md

```diff
@@ -0,0 +1,53 @@
+# Airside systems – Rule 10-9  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True     
+**Rule ID:** 10-9  
+ 
+**Rule Description:** Where multiple HVAC zones or residential spaces are combined into a single thermal block, baseline HVAC System Types 5 or 6 efficiencies shall be based on the cooling equipment capacity of a single floor.  
+
```

Comment:
I believe this RDS is related to possible use of multipliers. We discussed it with Jason a while back, and his posibition was that even if multipliers were used in the simulation, the RMR should include explicit listing of all zones, systems, etc. @JasonGlazer please chime in. I believe one of the outcomes of this discussion is the Zone.ID note that suggests that "No multipliers or floor multipliers are used with the Zone data group so each zone should be individually identified." I think question of multipliers was brought by one of the vendors, so I suggest revisiting it at the next RDS call and perhaps also the next schema WG call. 

If that's the case, we should tag any baseline HVAC systems of Type 5-8 that span multiple floors as an error, so the check wouldn't apply

Reply:
As discussed with Mike R per G3.1.1#4 "Floors with identical thermal blocks can be grouped for modeling purposes.", which means that a single unit could literally serve multiple floors (which I think is different than a multiplier situation).